x-pack/plugins/fleet/server/services/epm/elasticsearch/template/__snapshots__/template.test.ts.snap

```diff
@@ -56,6 +56,74 @@ exports[`EPM template tests loading base.yml: base.yml 1`] = `
 }
 `;
 
+exports[`EPM template tests loading cockroachdb_dynamic_templates.yml: cockroachdb_dynamic_templates.yml 1`] = `
+{
+  "properties": {
+    "cockroachdb": {
+      "properties": {
+        "status": {
+          "properties": {
+            "labels": {
+              "properties": {}
+            },
+            "*": {
```

Comment:
This static field shouldn't be here, right? This creates a mapping for a field called literally `cockroachdb.status.labels.*`.

Reply:
Yes you are correct, I just pushed a fix for that we do not create mappings is there only dynamic template for that path.